jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/SqlGenerationVisitor.java

```diff
@@ -73,10 +74,12 @@
 
     private final SqlDialect dialect;
     private final SqlGenerationContext context;
+    protected final SqlGenerationHelper helper;
 
     public SqlGenerationVisitor(final SqlDialect dialect, final SqlGenerationContext context) {
         this.dialect = dialect;
         this.context = context;
+        this.helper = new SqlGenerationHelper();
```

Comment:
This can be avoided if static. We could even simply add the static function directly to the SqlGenerationVisitor class, to avoid a generic "Helper" class with a random unrelated collection of helpers. Didn't think it through though.

Reply:
I changed the methods to static. I think the extra class makes sense though to keep the Generator class "clean".